### DISCUSSION TITLE
[🆘HELP🆘] Terminate transcribe process

### DISCUSSION THREAD

**Discussion by rudymohammadbali** [openai/whisper#1769] [Q&A]
Hello,

**I've reviewed the Whisper code, and it appears that there is currently no function available to cancel or terminate the transcribe process.**

I'm in urgent need of a method to halt the process, as I'm developing a GUI application where, upon clicking the Cancel button, the transcribing process should be stopped.

_Thank you in advance for your assistance._

**Comment by glangford** [comment]
run whisper in a subprocess, then kill the subprocess on cancel?

**Reply by rudymohammadbali** [reply]
How will it be done, could you write code here. And thanks for replaying!

**Reply by rudymohammadbali** [reply]
Also I have figured out that I need result object from whisper.transcribe() function. 

So using subprocess isn't going to work in my situation.
